src/styles/main.scss

```diff
@@ -1,3 +1,29 @@
+@import "./utils/_variables";
+@import "./utils/_reset";
+
+@import "./blocks/_header";
+@import "./blocks/_menu-header";
+@import "./blocks/_catalog";
+@import "./blocks/_card";
+@import "./blocks/_stars";
+
+@import url("https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap");
+
 body {
-  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: $main-color;
+  font-weight: 500;
+  line-height: 1.5;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  overflow: clip;
+  min-height: 100%;
+
+  &__main {
+    flex: 1 1 auto;
+  }
 }
```

Comment:
Let's keep the main.scss clean. Move body styles to normalize or reset file and for wrapper block let's create a separate file.